recipes/fiftyone-brain/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fiftyone-brain" %}
+{% set version = "0.16.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fiftyone-brain-{{ version }}.tar.gz
+  sha256: a3f52bf5b7be1516f7b5202b434d8e15cf60e570b1b9d13652a0ad9610864b67
+
+build:
+  skip: false
+  # skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy >=1.2.0
+    - scikit-learn
+
+test:
+  imports:
+    # - fiftyone.brain # would create a cyclic dependency
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/voxel51/fiftyone-brain
```

Comment:
This is a 404.

Reply:
Weird. It seems they changed the repo visibility to private. Using the pypi link instead.